filebeat/docs/inputs/input-journald.asciidoc

```diff
@@ -0,0 +1,232 @@
+possible.
+
+["source","yaml",subs="attributes"]
+----
+{beatname_lc}.inputs:
+- type: journald
+  id: iptables
+  include_matches:
+    - _TRANSPORT=kernel
+  processors:
+    - drop_event:
+        when.not.regex.message: '^iptables'
+----
+
+Each example adds the `id` for the input to ensure the cursor is persisted to
+the registry with a unique ID. The ID should be unique among journald inputs.
+If you don't specify and `id` then one is created for you by hashing
+the configuration. So when you modify the config this will result in a new ID
+and a fresh cursor.
+
+[id="{beatname_lc}-input-{type}-options"]
+==== Configuration options
+
+The `journald` input supports the following configuration options plus the
+<<{beatname_lc}-input-{type}-common-options>> described later.
+
+[float]
+[id="{beatname_lc}-id"]
+==== `id`
+
+An optional unique identifier for the input. By providing a unique `id` you can
+operate multiple inputs on the same journal. This allows each input's cursor to
+be persisted independently in the registry file.
+
+["source","yaml",subs="attributes"]
+----
+{beatname_lc}.inputs:
+- type: journald
+  id: consul.service
+  include_matches:
+    - _SYSTEMD_UNIT=consul.service
+
+- type: journald
+  id: vault.service
+  include_matches:
+    - _SYSTEMD_UNIT=vault.service
+----
+
+[float]
+[id="{beatname_lc}-paths"]
+==== `paths`
+
+A list of paths that will be crawled and fetched. Each path can be a directory
+path (to collect events from all journals in a directory), or a file path. If
+you specify a directory, {beatname_uc} merges all journals under the directory
+into a single journal and reads them.
+
+If no paths are specified, {beatname_uc} reads from the default journal.
+
+[float]
+[id="{beatname_lc}-backoff"]
+==== `backoff`
+
+The number of seconds to wait before trying to read again from journals. The
+default is 1s.
+
+[float]
+[id="{beatname_lc}-max-backoff"]
+==== `max_backoff`
+
+The maximum number of seconds to wait before attempting to read again from
+journals. The default is 60s.
+
+[float]
+[id="seek"]
+==== `seek`
+
+The position to start reading the journal from. Valid settings are:
+
+* `head`: Starts reading at the beginning of the journal. After a restart,
+{beatname_uc} resends all log messages in the journal.
+* `tail`: Starts reading at the end of the journal. After a restart,
+{beatname_uc} resends the last message, which might result in duplicates. If
+multiple log messages are written to a journal while {beatname_uc} is down,
+only the last log message is sent on restart.
+* `cursor`: On first read, starts reading at the beginning of the journal. After
+a reload or restart, continues reading at the last known position.
+
+When specified under `paths`, the `seek` setting applies to all journals under
+the configured paths. When specified directly under the +{beatname_lc}+
+namespace, the setting applies to all journals read by {beatname_uc}.
```

Comment:
Does this apply to Filebeat?

Reply:
No, those are not available. I removed this section and another similar one from the docs.